maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gitexe/src/main/java/org/apache/maven/scm/provider/git/gitexe/command/checkin/GitCheckInCommand.java

```diff
@@ -57,6 +58,14 @@
     extends AbstractCheckInCommand
     implements GitCommand
 {
+    private final Map<String, String> environmentVariables;
+
+    public GitCheckInCommand( Map<String, String> environmentVariables )
+    {
+        super();
+        this.environmentVariables = environmentVariables;
+    }
```

Comment:
There should be a zero-arg ctor too, no?

Reply:
I didn't consider this API, the only usage is from within `GitExeScmProvider` and that one only leverages the one with map.

Reply:
Alright, let's leave then as-is as long as it is consistent with the rest.